config/level1/Boss.py3

```diff
@@ -157,19 +155,19 @@ while True:
         toggle = player  # to avoid boring draws we start our push phase differently
 
     # We select the item we want to search
-    for i in items:
-        if(i[0] == quests[0]):
-            goalPos = Vector(i[1], i[2])
+    for item in items:
+        if(item.name == quests[0]):
+            goalPos = item.position
 
     # If it's a push turn
-    if(turn_type == 0):
+    if(turnType == 0):
         # We will alternate between vertical and horizontal push to shuffle the grid
         # and change the line or the column we want to push at each push
         if(toggle):
-            print("PUSH "+str(column) + " RIGHT")
+            print("PUSH " + str(column) + " RIGHT")
             column = (column + 1) % boardHeight
         else:
-            print("PUSH "+str(row) + " DOWN")
+            print("PUSH " + str(row) + " DOWN")
```

Comment:
You could also do either:

    print("PUSH", row, "DOWN")

or:

    print(f"PUSH {row} DOWN")

Reply:
c310020b6eb301bab436045923b322a0ed8371b5